src/config/db.js

```
@@ -0,0 +1,29 @@
+const { Sequelize } = require('sequelize');
+
+require('dotenv').config();
+
+const sequelize = new Sequelize(
+  process.env.DB_NAME || 'node_auth_db',
+  process.env.DB_USER || 'sofia',
+  process.env.DB_PASSWORD || 'mypassword',
```

While using environment variables is the correct approach, providing hardcoded fallback values for credentials like the database name, user, and password is a security risk. It's better to make these variables required and have the application fail to start if they are not set, rather than defaulting to potentially insecure values.